go.mod

```diff
@@ -79,19 +77,18 @@ require (
 
 require (
 	emperror.dev/errors v0.8.1 // indirect
-	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
+	github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161 // indirect
 	github.com/ansel1/merry v1.8.0 // indirect
-	github.com/ansel1/merry/v2 v2.2.0 // indirect
+	github.com/ansel1/merry/v2 v2.2.1 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/ceph/ceph-csi/api v0.0.0-20231227104434-06f9a98b7a83
+	github.com/ceph/ceph-csi/api v0.0.0-20241209162139-8ffd9f515a28
```

Comment:
@travisn are we still planning to leave this dependency at its old version and have the CSI team do this update?

Reply:
Discussion in #15128 is that we already covered everything csi recommended, so we will just keep it in this PR.